### ISSUE TITLE
added Doc Comments? Ja. / Made the auto job work: jein / Copy the files to the correct directory (even if the token at least works now. Thx4that btw....but): Nein!

### ISSUE THREAD

**Issue by thedizzle69** [thedizzle69/RetroRealmTTT#74]

**Comment by thedizzle69** [comment]
yeah but the comments are there and it more or less works. Still asking myself where the bools are...

**Comment by thedizzle69** [comment]
fuck this shit! I spent hours on a rollback because the javadoc file overwrote the main branch with the docs only. So all config regarding github were gone. I had to find a way to recover. Rebasing didnt help. Pushing wouldnt work as the github folders were gone. But one thing worked! Changing the damn default branch to a working one. Fetching it. Deleting the main branch. Make a new main branch and rebase the last working branch onto this new main branch. Switch back to this branch as default. And pulling a random new branch to test if the tests are working again. Also in the process I changed the problematic line and now the docs are going to the gh-pages branch and it seems they get finally get copied correctly to the docs directory. Maybe this is also because in the meanwhile i adapted all tokens.
